wdio/features/screen-objects/ImportFromSeedScreen.js

```diff
@@ -0,0 +1,36 @@
+import {
+  IMPORT_FROM_SEED_SCREEN_TITLE_ID,
+  IMPORT_FROM_SEED_SCREEN_SEED_PHRASE_INPUT_ID,
+  IMPORT_FROM_SEED_SCREEN_NEW_PASSWORD_INPUT_ID,
+  IMPORT_FROM_SEED_SCREEN_CONFIRM_PASSWORD_INPUT_ID,
+  IMPORT_FROM_SEED_SCREEN_SUBMIT_BUTTON_ID,
+} from '../testIDs/Screens/ImportFromSeedScreen.testIds';
+import Selectors from '../helpers/Selectors';
+
+class ImportFromSeed {
+  async verifyScreenTitle() {
+    await expect(await $(`~${IMPORT_FROM_SEED_SCREEN_TITLE_ID}`)).toBeDisplayed();
+  }
+
+  async typeSecretRecoveryPhrase(phrase) {
```

Comment:
Same. re: utilizing gesture methods in the gestures.js file. it should be

```suggestion
 async typeSecretRecoveryPhrase(phrase) {
    await Gestures.typeText(IMPORT_FROM_SEED_SCREEN_SEED_PHRASE_INPUT_ID, phrase);
  }

```